cwltool/pathmapper.py

```diff
@@ -191,6 +192,16 @@ def files(self) -> List[str]:
     def items(self) -> List[Tuple[str, MapperEnt]]:
         return list(self._pathmap.items())
 
+    def items_exclude_children(self) -> List[Tuple[str, MapperEnt]]:
```

Comment:
FYI @tetron @DailyDreaming @kellrott  about this proposed addition to PathMapper